lib/morpheus.gi

```diff
@@ -660,6 +660,8 @@ end);
 ##  dom   a set of elements on which automorphisms act faithful
 ##  aut   Subgroup of already known automorphisms
 ##  condition function that must return `true' on the homomorphism.
+##  setrun  If set to true approximate a run through sets by having the
+##          class indices increasing.
 ##
```

Comment:
Well, this is a technical parameter in a rather technical utility function. I don't think it makes sense to check this in the current test file format. (I can give you a 2 page function which uses it.)